INSTALL.md

```diff
@@ -23,6 +23,14 @@ recommended Perl interpreter as that is what the developers have tested, it
 supports the latest version of Perl, and includes all necessary Perl modules.
 It can coexist along side other interpreters.
 
+This also uses [ActiveTcl](https://www.activestate.com/products/tcl/), which is
```

Comment:
Totally get that we need some version of Tcl on Windows. I don't love that this introduces a non-open-source dependency.

Can we do a sniff test to see if other versions of Tcl work on Windows so we have a fall-back in case ActiveState changes something? eg https://www.tcl3d.org/bawt/download.html#tclpure

Reply:
This will require a bit of manipulation, but I THINK it should be okay @cpeel @windymilla. Long story short, even BAWT Tcl has this silly misconfigured tclConfig.sh problem, but in this case it puts something like:

`TCL_PACKAGE_PATH='{/C/BawtBuilds/TclDistribution/TclDistribution-8.6.12/Windows/x64/Release/Install/Tcl/lib}'` in incorrectly, rather than the installed default of C:\Tcl or whatever. I'm pretty sure I can just modify the same workaround I had going for ActiveTcl to work on this. I'll have to do it a few ways so that I can abstract away the `8.6.12` to work with whatever version including 8.7 alphas, and for x64 and 32bit to both work too. Shouldn't be too big a deal though. Assuming that works, I'll change the instructions to use this BAWT Tcl Batteries included, and we can decide whether to encourage 8.6 or let people go with 8.7 (I'll compare those too).

This seems like a much better solution. No accounts to log into, stays open source. I suppose I'll also want to document that the script is set up to clean up these variable names assuming this particular installation, the use of other installations means the regexes need to change.

Reply:
As a quick heads up -- I got it working with the tcl3d.org version, and am updating the instructions accordingly. The one remaining issue is for homebrew users who still can't install bwidget (https://github.com/Homebrew/homebrew-core/pull/120817). I might try vendoring in BWidget (and limiting its usage to Mac users only), with the expectation that once it hits homebrew, we can remove it.

Reply:
Additionally, the 8.7 alphas do not work at the moment. I can make the regexp generic enough to roll with it, but the Tcl code itself that some of these packages are calling aren't yet compatible.

Reply:
Fixed with ad76a9b.